data/racecar/ground.urdf

```diff
@@ -0,0 +1,31 @@
+<robot name="ground">
+  <link name="world"/>
+  <link name="ground_link">
+    <inertial>
+      <mass value="100.0"/>
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <inertia ixx="1.0" ixy="0" ixz="0" iyy="1.0" iyz="0" izz="1.0"/>
+    </inertial>
+    <visual>
+      <origin xyz="0 0 -0.525" rpy="0 0 0"/>
```

Comment:
Let's change the offset to `0 0 -0.025` so that the surface of the ground to be at zero in the z-axis.